src/redis-cli.c

```diff
@@ -3028,7 +3028,8 @@ static void usage(int err) {
 "  --askpass          Force user to input password with mask from STDIN.\n"
 "                     If this argument is used, '-a' and " REDIS_CLI_AUTH_ENV "\n"
 "                     environment variable will be ignored.\n"
-"  -u <uri>           Server URI.\n"
+"  -u <uri>           Server URI (redis://user:password@host:port/dbnum)\n"
+"                     For connection strings without a username, use default:password@host:port.\n"
```

Comment:
Adding some more details and trying to keep the output not more than 80 characters wide, the default width of a terminal.

```suggestion
"  -u <uri>           Server URI on format redis://user:password@host:port/dbnum\n"
"                     User, password and dbnum are optional. For authentication\n"
"                     without a username, use 'default' as the username.\n"
"                     For TLS, use 'rediss' as the protocol.\n"
```

Reply:
@zuiderkwast Thank you for providing guidance on the 80 characters wide line spacing. I have updated the code based on the feedback you provided.